<launch>

    <rosparam command="load" file="$(find upo_path_tracker)/yaml/navigation.yaml"/>
    <node pkg="upo_path_tracker" type="simple_path_tracker_node" name="nav_node" output="screen"/>

</launch>
